vehicle_4ws:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100

  #Controllers --------------------------------------
  front_right_steering_position_controller:
    type: effort_controllers/JointEffortController
    joint: front_right_steering
    pid: {p: 100.0, i: 0.01, d: 0.01}
  front_left_steering_position_controller:
    type: effort_controllers/JointEffortController
    joint: front_left_steering
    pid: {p: 100.0, i: 0.01, d: 0.01}
  front_right_wheel_velocity_controller:
    type: effort_controllers/JointEffortController
    joint: front_right_wheel
    pid: {p: 100.0, i: 0.01, d: 0.01}
  front_left_wheel_velocity_controller:
    type: effort_controllers/JointEffortController
    joint: front_left_wheel
    pid: {p: 100.0, i: 0.01, d: 0.01}
  rear_right_wheel_velocity_controller:
    type: effort_controllers/JointEffortController
    joint: rear_right_wheel
    pid: {p: 100.0, i: 0.01, d: 0.01}
  rear_left_wheel_velocity_controller:
    type: effort_controllers/JointEffortController
    joint: rear_left_wheel
    pid: {p: 100.0, i: 0.01, d: 0.01}
